tracing/src/main/java/com/palantir/tracing/Trace.java

```diff
@@ -136,6 +136,8 @@ final String getTraceId() {
         return traceId;
     }
 
+    abstract Optional<String> getRequestId();
```

Comment:
We still store the request id so that it can be restored when a trace is set. However this getter is not exposed to clients of `Tracers` / `Tracer`